containers/kubernetes/how-to/properly-upgrade-kubernetes-version.mdx

```diff
@@ -0,0 +1,96 @@
+---
+meta:
+title: How to properly upgrade your Kubernetes Kapsule cluster
+description: This page explains how to properly upgrade your Kubernetes Kapsule cluster
+content:
+h1: How to properly upgrade your Kubernetes Kapsule cluster
+paragraph: This page explains how to upgrade your Kubernetes Kapsule cluster
+tags: kubernetes kubernetes-kapsule kapsule cluster
+dates:
+categories:
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+    - You have a working [CLI](https://github.com/scaleway/scaleway-cli) with your credentials set up
+    - You have a Kubernetes Kapsule cluster which is not running the latest k8s version available on Kapsule API and
+    you're willing to upgrade it.
+</Message>
+
+First, we'll make sure your workload can be properly handled by the newer Kapsule version. We have a compatibility matrix
+for different components and your current cluster may use components that are deprecated or not available on the new version.
+Please see our [version policy](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy/).
+
+For each version upgrade, the client is encouraged to read kubernetes [changelog](https://kubernetes.io/releases/).
+
+## Checking which components must be changed
+
+An easy way to visualize this is with the command `scw k8s version list`.
+e.g.
+```
+1.25.9   [cilium calico kilo]                [none]                 [containerd]
+1.24.13  [cilium calico weave flannel kilo]  [none]                 [containerd crio]
+1.23.17  [cilium calico weave flannel kilo]  [none nginx traefik2]  [containerd crio docker]
+```
+
+### Ingresses
+
+Managed Ingresses are deprecated since the minor release 1.24, we provide the feature EasyDeploy instead to set up an ingress controller.
```

Comment:
```suggestion
Managed ingress controllers are deprecated since the minor release 1.24, we provide the feature EasyDeploy instead to set up an ingress controller.
```